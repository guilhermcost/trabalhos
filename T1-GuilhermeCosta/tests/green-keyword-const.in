int main(void) {
 int CONST i = 1000;
}
