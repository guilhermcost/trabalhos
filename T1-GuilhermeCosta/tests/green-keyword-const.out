(1,KEY,"int")
(1,ID,"main")
(1,SYM,"(")
(1,KEY,"void")
(1,SYM,")")
(1,SYM,"{")
(2,KEY,"int")
(2,ID,"CONST")
(2,ID,"i")
(2,SYM,"=")
(2,NUM,"1000")
(2,SYM,";")
(3,SYM,"}")
